warden/warden_daemon/src/app.rs

```diff
@@ -0,0 +1,112 @@
+use super::cli::Cli;
+use super::client_handler::client_command_handler::ClientHandler;
+use super::fabric::realm_fabric::RealmManagerFabric;
+use super::fabric::warden_fabric::WardenFabric;
+use super::managers::warden::RealmCreator;
+use super::managers::warden::Warden;
+use super::socket::unix_socket_server::{UnixSocketServer, UnixSocketServerError};
+use super::socket::vsocket_server::{VSockServer, VSockServerConfig, VSockServerError};
+use anyhow::Error;
+use log::{debug, error, info};
+use std::sync::Arc;
+use std::time::Duration;
+use tokio::select;
+use tokio::signal::unix::{signal, SignalKind};
+use tokio::spawn;
+use tokio::sync::Mutex;
+use tokio::task::JoinHandle;
+use tokio_util::sync::CancellationToken;
+
+pub struct App {
```

Comment:
Minor: Why App and not something like Daemon?

Reply:
Done